### PR TITLE
Requesting for new coin type (TTNC)

### PR DESCRIPTION
**Pull request by blinc-usa**

### CODE REVIEW

File: slip-0044.md [hunk 1]
```diff
@@ -1352,6 +1352,8 @@ All these constants are used as hardened derivation.
 | 608589380  | 0xa4465644                    | FVDC    | ForumCoin                         |
 | 1179993420 | 0xc655454c                    |         | Fuel                              |
 | 1179993421 | 0x80233434                    | TTNC    | TakeTitan                         |
+| 1179993441 | 0x80233444                    | QFS     | Qfs                               |
```
Comment (prusnak) [inline slip-0044.md:1355]: 1. keep the list sorted
2. make sure the dec values match the hex values
3. please fix also TTNC value which is wrong

**Review by prusnak — CHANGES_REQUESTED** [review]

File: slip-0044.md [hunk 2]
```diff
@@ -1351,6 +1351,9 @@ All these constants are used as hardened derivation.
 | 20240430   | 0x8134d82e                    | NLK     | NuLinkCoin                        |
 | 608589380  | 0xa4465644                    | FVDC    | ForumCoin                         |
 | 1179993420 | 0xc655454c                    |         | Fuel                              |
+| 1179993431 | 0x464642F7                    | MTGBP   | MTGBP                             |
+| 1179993441 | 0x46464301                    | QFS     | Qfs                               |
+| 1179993421 | 0x464642ED                    | TTNC    | TakeTitan                         |
```
Comment (prusnak) [inline slip-0044.md:1356]: 1. This is not sorted according to value
2. Do not user upper-case letters in hex values
3. Hex values do not match the decimal values

Comment (blinc-usa) [reply slip-0044.md]: @prusnak  Please Check now I have made the changes according to your comments .

Comment (prusnak) [reply slip-0044.md]: No, the point 3 is not addressed

1179993421 is 0x4655454d not 0x464642ed for example

Comment (blinc-usa) [reply slip-0044.md]: @prusnak Thanks for updating me on the error.Now please check ..!!!

Comment (prusnak) [reply slip-0044.md]: the hex numbers are still wrong and do not match the numbers in the first column

**Pull request merged**